axis.go

```diff
@@ -39,3 +39,48 @@ func (keys AxisKeyPair) Value() float32 {
 
 	return AxisNeutral
 }
+
+const (
+	// AxisMouseVert is vertical mouse axis
+	AxisMouseVert = 0
+	// AxisMouseHori is vertical mouse axis
+	AxisMouseHori = 1
+)
+
+// AxisMouse is an axis for a single x or y component of the Mouse. The value returned from it is
+// the delta movement, since the previous call and it is not constrained by the AxisMin and AxisMax values.
+type AxisMouse struct {
+	// direction is the value storing either AxisMouseVert and AxisMouseHori. It determines which directional
+	// component to operate on.
+	direction int
```

Comment:
Let's not make this an `int`, but creat something like `type AxisMouseDirection uint`?  Same for lines 45 and 47 above (they would be of that new type)